renderapi/render.py

```diff
@@ -2,9 +2,10 @@
 import logging
 import os
 import requests
-from .utils import defaultifNone, NullHandler, fitargspec
+from .utils import defaultifNone, NullHandler, fitargspec, get_json
 from .errors import ClientScriptError, RenderError
 from decorator import decorator
+from six.moves import input
```

Comment:
we can reclaim a bunch of code coverage by fixing how this works in the coveragerc.

currently it does not cover blocks which include raw_input.  we could either import this input as raw_input and keep the old code 
 or modify the coveragerc to exclude input lines (which we probably need to be more careful about)